examples/Detoxify.ipynb

```diff
@@ -223,6 +285,31 @@
     "tmarco.load_models([\"trustyai/gminus\", \"trustyai/gplus\"])"
    ]
   },
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "To use local expert/anti-expert models with TMaRCo, we need to have them in a local storage, accessible to TMaRCo, as previously.\n",
```

Comment:
Can we change this to:
"To use local expert/anti-expert models with TMaRCo, we need to have them in a local storage that is accessible to TMaRCo, as previously."

for clarity?